ehrql/tables/beta/tpp.py

```diff
@@ -231,80 +247,180 @@ class clinical_events(EventFrame):
 
 @table
 class ec(EventFrame):
+    """
+    Emergency care admissions data is provided via the NHS Secondary Uses Service.
+
+    This table gives core details of admissions.
+    """
+
     ec_ident = Series(
         int,
         constraints=[Constraint.NotNull()],
-        description="TODO",
+        description="Unique identifier used across the EC tables.",
```

Comment:
Could replace `unique identifier` with `unique identifier for the activity/spell/attendance/appointment` to be clear it's not a patient identifier ?